src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEClassificationList.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * @author Walker Colina
+ */
+package org.umssdiplo.automationv01.core.managepage.ppe;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.umssdiplo.automationv01.core.managepage.BasePage;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+public class PPEClassificationList extends BasePage {
```

Comment:
this page should have a constructure where it should wait for something in specia.